#!/usr/bin/env python
PACKAGE = "real_time_simulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wind_speed",    int_t,   0, "The wind speed", 10,  0, 100)
gen.add("wind_direction", int_t,  0, "The wind direction",    45, 0,   359)

exit(gen.generate(PACKAGE, "Wind", "Wind"))